### PR TITLE
Update documentation, remove endpoint not yet served

### PR DESCRIPTION
**Pull request by brookslogan**
@keanmingtan I went ahead and made a PR for your changes, as we did some epidatr testing at a tooling meeting off of your branch, and might have comments to add on particular lines (PR is the best place I think).  Do you have additional work locally that should be added?

### CODE REVIEW

File: R/endpoints.R [hunk 1]
```diff
@@ -79,20 +79,25 @@ pvt_cdc <- function(auth, epiweeks, locations) {
...
 #' call <- covid_hosp_facility_lookup(state = "fl")
 #' fetch_csv(call)
 #' }
-#' @param state optional character state
-#' @param ccn optional character ccn
-#' @param city optional character city
-#' @param zip optional numeric zip code
-#' @param fips_code optional numeric fips code
-#' @return an instance of epidata_call
+#' @param state A two-letter character string state abbreviation.
```
Comment (brookslogan) [inline R/endpoints.R:91]: With fix from #74, maybe this could be a character vector.

Comment (brookslogan) [reply R/endpoints.R:91]: Ditto for probably every instance of "character string" below.

**Review by brookslogan — COMMENTED** [review]
I removed the history file stuck in here and updated the `.gitignore`; once you pull the `.gitignore` update, git shouldn't try to add it back anymore.  Added a comment, but haven't given it a full pass.  Dmitry, would you be able to finish the review?

**Comment by dshemetov** [comment]
Yup, I'll look at it on Monday.

**Comment by keanmingtan** [comment]
update documentation for gft, nidss_dengue, nidss_flu, nowcast, fluview, flusurv, ecdc_ili

**Comment by keanmingtan** [comment]
Also have some questions about the following:  
NIDSS dengue stop updating from 2018? 
NIDSS flu data stop updating from 2018? 
Google flu trend stop updating from 2015? 
EDCD data stop updating from 2020?

**Comment by dshemetov** [comment]
Hm, it occurs to me that we should probably build and host the documentation for this package and integrate CI into it. Would make documentation easier to review.

**Comment by dshemetov** [comment]
I have made a number of edits in #77. Many small changes.

I think with the changes there, this PR should be ready to merge into dev. I figure 
- we will need do another documentation pass once API key changes are introduced
- we should focus our attention there, get this merged, start using the hosted docs, and begin collecting docs TODOs in an issue for another docs pass later

**Review by dshemetov — APPROVED** [review]

**Pull request merged**